bigbluebutton-html5/imports/ui/components/actions-bar/emoji-select/component.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
This file is supposed to be shown as "renamed", isn't it?
So what's supposed to happen is:
you change the name of the folder, and file "folder1/aaa.js" becomes  "folder2/aaa.js->
commit both files, deleted and new one ->
git knows that it's the same file, but it was renamed ->
you made your changes ->
they are shown as a diff at the end

But in this case you renamed the folder -> made changes -> committed.
As a result git didn't recognize the similarities between the files and showed it as a completely new file.
Thus file history was destroyed, which is not good.